docs/filter.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Filter
```

Comment:
The `Filter` and `Label` pages are extra features on `phylum analyze`. I think we should either link to these pages from `analyzing-dependencies.md` or maybe even merge the 3 pages into 1.